packages/ia-components/sandbox/jwplayer-comp/.babelrc

```diff
@@ -1,23 +1,25 @@
 {
-    "presets": [
-        [
-          "env", {
-            "targets": {
-              "node": "current"
-            }
-          }
-        ],
-        "react"
+  "presets": [
```

Comment:
is this file needed? not sure what `jwplayer-comp` directory is..

Reply:
@iisa, the `JWPlayerContainer` component is being used by `packages/ia-components/sandbox/theatres/audio-1/audio-1-br-nav-wav.js` and `packages/ia-components/sandbox/theatres/audio-1/audio-1.js`. 
The plugins and presets specified in `.babelrc`  of `jwplayer-comp` are already specified in the `.babelrc` of `ia-components`. So it is perfectly safe to delete the inner `.babelrc` file.